docs/FAQ.md

```diff
@@ -45,8 +45,8 @@ version with a message that looks something like this:
 The full RAPIDS Accelerator and cudf build properties are logged at `INFO` level in the
 Spark driver and executor logs with messages that are similar to the following:
 ```
-21/04/14 17:20:20 INFO RapidsExecutorPlugin: RAPIDS Accelerator build: {version=0.5.0-SNAPSHOT, user=jlowe, url=, date=2021-04-14T22:12:14Z, revision=79a5cf8acd615587b2c7835072b0d8b0d4604f8b, cudf_version=0.19-SNAPSHOT, branch=branch-0.5}
-21/04/14 17:20:20 INFO RapidsExecutorPlugin: cudf build: {version=0.19-SNAPSHOT, user=, date=2021-04-13T08:42:40Z, revision=a5d2407b93de444a6a7faf9db4b7dbf4ecbfe9ed, branch=HEAD}
+21/04/14 17:20:20 INFO RapidsExecutorPlugin: RAPIDS Accelerator build: {version=0.5.0, user=jlowe, url=, date=2021-04-14T22:12:14Z, revision=79a5cf8acd615587b2c7835072b0d8b0d4604f8b, cudf_version=0.19, branch=branch-0.5}
+21/04/14 17:20:20 INFO RapidsExecutorPlugin: cudf build: {version=0.19, user=, date=2021-04-13T08:42:40Z, revision=a5d2407b93de444a6a7faf9db4b7dbf4ecbfe9ed, branch=HEAD}
```

Comment:
Note that if we manually update this output, the git commit hashes shown on the same lines will not match those versions.

Reply:
I reverted this change.  I can update the text for the gh-pages branch when we have a non-snapshot version of the release.